lib/src/component/resize_sensor.dart

```diff
@@ -161,7 +181,9 @@ class ResizeSensorComponent extends UiComponent<ResizeSensorProps> with _SafeAni
       (Dom.div()..style = _collapseSensorChildStyle)()
     );
 
-    var resizeSensor = (Dom.div()
+    // If we have the Resize Observer API, the resize sensor is a no-op
+    // otherwise, wire up the div, expand and collapse sensor just as before
+    var resizeSensor = ResizeObserver.supported ? null : (Dom.div()
```

Comment:
`ResizeObserver.supported` will never be true here, right?

Reply:
It **can** be true here. In the case where we are using the resize observer path, Ive changed the behavior in the `componentDidMount` and `render` methods. In render, it will still render its children but not the extra expandSensor and collapseSensor elements.